### ISSUE TITLE
Should there be no penalty for DEV to refuse issue with impediment?

### ISSUE THREAD

**Issue by llorllale** [zerocracy/datum#257]
Scenario: as a `DEV` I am assigned a job, but then I'm blocked by some other reported issue, so I put it on `waiting`. The blocking issue has been acknowledged but has not been assigned a performer, so it's going nowhere for the time being. 

My job is now an opportunity cost (I could be working on something else), but I can't `refuse` because I'll then be penalized with -15 points.

What should I do?

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by llorllale** [comment]
I think I've answered my own question: we should use [puzzles](http://www.yegor256.com/2010/03/04/pdd.html) and [cut corners](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
Job `gh:zerocracy/datum#257` is not assigned, can't get start time

**Comment by 0crat** [comment]
Job [#257](https://github.com/zerocracy/datum/issues/257) was not in scope
